### ISSUE TITLE
Segmented 3DGS

### ISSUE THREAD

**Issue by Lizhinwafu** [Jumpat/SegAnyGAussians#47]

**Comment by Jumpat** [comment]
The rendered mask has some difference with the segmented Gaussians. This is because that we use a little threshold on the rendered mask.

**Comment by Jumpat** [comment]
Hi. The 0.5 in the [notebook](https://github.com/Jumpat/SegAnyGAussians/blob/main/prompt_segmenting.ipynb)

> plt.imshow(rendered_mask_pkg['mask'].squeeze().detach().cpu() >= 0.5)
